tests/forms_tests/widget_tests/test_dateinput.py

```diff
@@ -0,0 +1,49 @@
+from datetime import date
+
+from django.forms import DateInput
+from django.test import override_settings
+from django.utils import translation
+
+from .base import WidgetTest
+
+
+class DateInputTest(WidgetTest):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.widget = DateInput()
```

Comment:
I wonder if this initialization needs to be done in `setUpClass()` or if doing it at the class-level would work?
```
class DateInputTest(WidgetTest):
    widget = DateInput()
````